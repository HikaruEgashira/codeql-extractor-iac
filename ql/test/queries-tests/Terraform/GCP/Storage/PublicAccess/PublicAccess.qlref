security/Terraform/GCP/Storage/PublicBucket.ql