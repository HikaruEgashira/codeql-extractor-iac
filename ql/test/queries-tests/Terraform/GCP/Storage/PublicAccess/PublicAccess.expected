| storage.tf:7:1:10:1 | resource google_storage_bucket bucket | Public Storage Bucket. |
